flink-staging/flink-language-binding/flink-language-binding-generic/src/main/java/org/apache/flink/languagebinding/api/java/common/streaming/Sender.java

```diff
@@ -253,6 +257,18 @@ public ByteBuffer serialize(T value) {
 		public abstract void serializeInternal(T value);
 	}
 
+	private class CustomTypeSerializer extends Serializer<CustomTypeWrapper> {
+		public CustomTypeSerializer() {
+			super(0);
+		}
+		@Override
+		public void serializeInternal(CustomTypeWrapper value) {
+			byte[] bytes = value.getData();
+			buffer = ByteBuffer.allocate(bytes.length);
```

Comment:
I guess you can use `ByteBuffer.wrap()` to avoid allocating an additional byte array for every custom type?